remix-debug/src/solidity-decoder/internalCallTree.js

```diff
@@ -297,8 +297,9 @@ function addParams (parameterList, tree, scopeId, states, contractName, sourceLo
     if (stackDepth >= 0) {
       let location = typesUtil.extractLocationFromAstVariable(param)
       location = location === 'default' ? 'memory' : location
-      tree.scopes[scopeId].locals[param.attributes.name] = {
-        name: param.attributes.name,
+      const attributesName = param.attributes.name === '' ? `$${inputParam}` : param.attributes.name
```

Comment:
`$${inputParam}` ?

Reply:
This kind of a manual convention to denote anonymous params as with ' ', more than one parameters were getting overwritten